kloppy/domain/models/pitch.py

```diff
@@ -21,6 +21,22 @@ class PitchDimensions:
     x_per_meter: float = None
     y_per_meter: float = None
 
+    @property
+    def length(self) -> float:
+        return (
+            (self.x_dim.max - self.x_dim.min) / self.x_per_meter
+            if self.x_per_meter is not None
```

Comment:
Please remove the ‘is not None’. It should cover both ‘None’ and 0

Reply:
Sure you're right, I should take into account both instead of one. Do you prefer that it throws and error when it's None or 0, or that both cases are handled ?

Reply:
When you remove “is not None” it’s solved for both cases. No need to throw exceptions

Reply:
Yes sure, but it will throw an error (division by zero).
I'll make the change this week

Reply:
Zero will evaluate to False

Reply:
In fact it fails if I remove the "is not None" condition:

```
   def length(self) -> float:
>       return (self.x_dim.max - self.x_dim.min) / self.x_per_meter
E       TypeError: unsupported operand type(s) for /: 'int' and 'NoneType'
```

I think we should have a condition to be sure it's properly handled.
Maybe I did not understood what you meant ?

Reply:
Ah! I understand the confusion. I was unclear.

What I meant is that both `None` and `0` are falsy ( https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/ ). So `X if condition else Y` will cover both `None` and `0`

Reply:
Ah! Alright so that it what I was thinking at first: add a condition on 0 and not remove the condition on None.

something like
` if (self.x_per_meter is not None) & (self.x_per_meter != 0)`

Reply:
that can be simplified to `if self.x_per_meter` :-)

Reply:
True. That means I now understand what you were saying at the beginning (and that I should rest)